plugins/jira-dashboard/src/hooks/useJiraUserIssues.ts

```diff
@@ -0,0 +1,21 @@
+import useAsync from 'react-use/lib/useAsync';
+import { JiraDashboardApi } from '../api';
+import { Issue } from '@axis-backstage/plugin-jira-dashboard-common';
+
+export function useJiraUserIssues(
+  maxResults: number,
+  jiraDashboardApi: JiraDashboardApi,
+): {
+  data: Issue[] | undefined;
+  loading: boolean;
+  error: Error | undefined;
+} {
+  const {
+    value: data,
+    loading,
+    error,
+  } = useAsync(async (): Promise<any> => {
```

Comment:
We can remove the cast here?  

```
  const {
    value: data,
    loading,
    error,
  } = useAsync(async () => {
    return jiraDashboardApi.getLoggedInUserIssues(maxResults);
  }, [jiraDashboardApi]);
  return { data, loading, error };
}
```

Reply:
Code updated. Just a copy of `useJira` hook :-).